### PR TITLE
475 bug fix cw address

### PR DESCRIPTION
**Pull request by Orta21**
Ref. metriport/metriport-internal#475

Ref: 475

### Dependencies

None

### Description

Address schema bug needed to be mostly optional

### Release Plan

- [ ] ASAP

### CODE REVIEW

File: packages/packages/api/src/medical/models/common/address.ts [hunk 1]
```diff
@@ -2,12 +2,12 @@ import { z } from "zod";
...
   addressLine2: z.string().optional().nullable(),
-  city: z.string().min(1),
-  state: usStateSchema,
-  zip: z.string().length(5),
-  country: z.string().min(1),
+  city: z.string().optional(),
+  state: usStateSchema.optional(),
+  zip: z.string(),
+  country: z.string().optional(),
 });
```
Comment (Orta21) [inline packages/packages/api/src/medical/models/common/address.ts:12]: If this looks fine ill have to deploy a new version and add commit to the PR

File: packages/packages/api/src/medical/models/common/address.ts [hunk 2]
```diff
@@ -2,12 +2,12 @@ import { z } from "zod";
...
+  addressLine1: z.string().optional(),
   addressLine2: z.string().optional().nullable(),
-  city: z.string().min(1),
-  state: usStateSchema,
-  zip: z.string().length(5),
-  country: z.string().min(1),
+  city: z.string().optional(),
+  state: usStateSchema.optional(),
+  zip: z.string(),
+  country: z.string().optional(),
```
Comment (Goncharo) [inline packages/packages/api/src/medical/models/common/address.ts:10]: Few things to consider here:

- We're using `addressSchema` for orgs/facilities as well, for which those fields should not be optional (ie they're required to make an org in CW).
- Will this break our form validation for patients?
- If making this change to our models, we'll need to update the docs accordingly - ie we use [this snippet](https://github.com/metriport/metriport/blob/ed3f188aa9a463eda21a379c34f1279008ac29cc/docs/_snippets/address.mdx#L1-L26) in our docs.

File: packages/packages/api/src/medical/models/common/address.ts (hunk 1 above)
Comment (Orta21) [reply packages/packages/api/src/medical/models/common/address.ts:12]: I had made more of the address schemas optional but realized the only one really affecting this is this one as the other ones are our internal schemas

**Comment by leite08** [comment]
Gotta update API's address schema as well, right?

**Comment by Orta21** [comment]
@leite08 I dont think so because it seems like the addressSchema on api is just for our db which we dont want to keep optional right?

**Review by Goncharo — CHANGES_REQUESTED** [review]
:

File: packages/packages/api/src/medical/models/common/address.ts (hunk 2 above)
Comment (Goncharo) [reply packages/packages/api/src/medical/models/common/address.ts:10]: Perhaps we should just fix the sandbox payload for now, and tackle the optional field change in a separate issue?

**Review by leite08 — APPROVED** [review]

**Review by Goncharo — APPROVED** [review]
🚢

**Pull request merged**